### PR TITLE
Removed obsolete and unnecessary files.

### PR DESCRIPTION
**Pull request by kc284**
Html for the scripts does not need to be in source control as it can be
autogenerated using various tools (e.g. pygmentize, enscript etc; the scripts
are even automatically highlighted on github).

Signed-off-by: Konstantina Chremmou konstantina.chremmou@citrix.com

### CODE REVIEW

**Comment by robhoes** [comment]
Note: I still need to review this, but in any case, we cannot put this in the master branch at this point in time.

**Comment by lindig** [comment]
I have marked this blocked because we cannot merge to trunk-ring3 at the moment.

**Comment by jonludlam** [comment]
Over to @robhoes to review :-)

File: scripts/examples/bash-cli/README [hunk 1]
```diff
@@ -1,49 +1,28 @@
...
-
-   This file describes the XE CLI + bash examples included in the [1]Software
-   Development Kit (SDK) for the [2]XenServer 4.0 product family.
-
 XE CLI + bash examples
 
-  install-debian
-
-   [3][ browse ] [4][ download ]
+ [install-debian][1]
```
Comment (robhoes) [inline scripts/examples/bash-cli/README]: Is this supposed to be markdown? I don't think the syntax is right.

File: scripts/examples/bash-cli/README [hunk 2]
```diff
@@ -1,49 +1,28 @@
...
-   3. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/install-debian.html
-   4. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/install-debian
-   5. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/suspend-resume.html
-   6. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/suspend-resume
-   7. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/clone-vms.html
-   8. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/clone-vms
-   9. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/move-management-to-bond.html
-  10. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/move-management-to-bond
+
+[1]: install-debian
```
Comment (robhoes) [inline scripts/examples/bash-cli/README]: Is this useful?

File: ocaml/idl/sdk-README.html [hunk 3]
```diff
@@ -1,110 +0,0 @@
-<html>
```
Comment (robhoes) [inline ocaml/idl/sdk-README.html]: I think that removing this file will break some targets in the ocaml/idl/OMakefile (e.g. noarch-install).

Comment (robhoes) [inline ocaml/idl/sdk-README.html]: Also the other html files in ocaml/idl: do they have replacements somewhere? Or is that not needed?

File: scripts/examples/bash-cli/README (hunk 1 above)
Comment (kc284) [inline scripts/examples/bash-cli/README]: It's markdown recognisable by github if the file has .md extension - which I've just realised this file does not have, so thanks for pointing it out, I'll rename it.

File: scripts/examples/bash-cli/README (hunk 2 above)
Comment (kc284) [inline scripts/examples/bash-cli/README]: Apologies, I don't understand your question. Are you asking whether this particular example is useful or whether it's useful to have a link to the example in the readme file?

File: ocaml/idl/sdk-README.html (hunk 3 above)
Comment (kc284) [inline ocaml/idl/sdk-README.html]: I believe these files used to be needed for the content on the old community page, but they're out of date and I don't think they're currently published anywhere. In any case I'll have another look to ensure the changes are not breaking anything.

**Comment by robhoes** [comment]
> The old name sdk-install was giving the impression they were called from the
homonymous rule further up the folder hierarchy, while they are actually a
prerequisite of noarch-install.

But I think it was called from _both_. I.e. from `make sdk-install` as well. No?

**Comment by robhoes** [comment]
It looks to me, however, that the sdk-install make target is completely unused, because it is not referenced by the spec file.

**Comment by kc284** [comment]
I've removed it. Now you can resume the review, I've updated the label :)

**Comment by kc284** [comment]
I think all those targets should have been removed when the SDK VM and the community site went, but they were left behind because it was not obvious what was needed for the docs.

**Comment by robhoes** [comment]
Fantastic ✨

**Pull request merged**